lib/evmone/advanced_instructions.cpp

```diff
@@ -269,6 +269,7 @@ constexpr std::array<instruction_exec_fn, 256> instruction_implementations = [](
     table[OP_DATALOADN] = op_undefined;
     table[OP_DATASIZE] = op_undefined;
     table[OP_DATACOPY] = op_undefined;
+    table[OP_JUMPF] = op_undefined;
```

Comment:
requires a similar addition in `legacy_cost_tables` in `baseline_instruction_table.cpp`?

Reply:
Right, good catch, thanks. Fixed and added a test.